addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -631,7 +631,7 @@
             </t>
             <span t-if="channel.can_publish" class="d-none d-md-flex">
                 <a t-if="slide.slide_category == 'article'" class="mx-2 o_text_link text-primary o_not_editable" target="_blank" t-attf-href="/slides/slide/#{slug(slide)}?enable_editor=1" title="Edit"><span class="fa fa-pencil"/></a>
-                <a t-else="" class="mx-2 o_text_link text-primary o_not_editable" target="_blank" t-attf-href="/odoo/action-{slide_action}/{slide.id}" title="Edit in backend"><span class="fa fa-pencil"/></a>
+                <a t-else="" class="mx-2 o_text_link text-primary o_not_editable" target="_blank" t-attf-href="/odoo/slide.slide/#{slide.id}" title="Edit in backend"><span class="fa fa-pencil"/></a>
```

Comment:
Hello @thsh-odoo :slightly_smiling_face: 
The url has been updated last week here:
https://github.com/odoo/odoo/commit/9d92a90ec2b0adae3f749da7a7b8582744ec492f#diff-fc5a5d1cec9d16637120c32f791f07a55bd749437887a4858d21d6ed98d2502bR634 
But is still failing because the slide_action doesn't exist.
Could you just replace the action with `slide.slide` and keep the double brackets around the slide.id ? :slightly_smiling_face: 
Thanks!